domain/tokens.go

```diff
@@ -3,9 +3,10 @@ package domain
 // Token represents the token's domain model
 type Token struct {
 	// HumanDenom is the human readable denom.
-	HumanDenom string `json:"human_denom"`
+	HumanDenom string `json:"symbol"`
 	// Precision is the precision of the token.
-	Precision int `json:"precision"`
+	Precision int `json:"decimals"`
 	// IsUnlisted is true if the token is unlisted.
-	IsUnlisted bool `json:"is_unlisted"`
+	IsUnlisted  bool   `json:"preview"`
+	CoingeckoID string `json:"coingeckoId"`
```

Comment:
Just realized that this would be a breaking change.

For example, it would break our new integration tests. I don't think there are any other clients so we can simply fix our integration tests.

Could you please confirm that the tests continue to work by running `e2e-run-dev`?

Reply:
https://github.com/osmosis-labs/sqs/blob/5062373fcf18dd144236a9c1ae4311d5be21a1b0/tests/setup.py#L286

Reply:
Addressed in https://github.com/osmosis-labs/sqs/pull/210/commits/170cd24df7f97a367ab4d842f863806f59e61bb9